app/actions/websocket/index.ts

```diff
@@ -27,21 +27,25 @@ import {handleUserRoleUpdatedEvent, handleTeamMemberRoleUpdatedEvent, handleRole
 import {handleLeaveTeamEvent, handleUserAddedToTeamEvent, handleUpdateTeamEvent} from './teams';
 import {handleUserUpdatedEvent, handleUserTypingEvent} from './users';
 
-import type {Model} from '@nozbe/watermelondb';
+// Remove once 5.37 is no longer supported
+const alreadyConnected = new Set<string>();
 
 export async function handleFirstConnect(serverUrl: string) {
-    const database = DatabaseManager.serverDatabases[serverUrl]?.database;
-    if (!database) {
+    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
+    if (!operator) {
         return;
     }
-    const config = await queryConfig(database);
-    const lastDisconnect = await queryWebSocketLastDisconnected(database);
-    if (lastDisconnect && config.EnableReliableWebSockets !== 'true') {
-        doReconnect(serverUrl);
+    const config = await queryConfig(operator.database);
+    const lastDisconnect = await queryWebSocketLastDisconnected(operator.database);
+
+    // Remove once 5.37 is no longer supported
```

Comment:
Ticket number?

Reply:
There is no ticket for this and no harm to leave the code, it just serves as a reminder

Reply:
Out of the scope of the ticket, but should we follow some pattern so we can come back to these (or create help wanted ticket) any time we push the ESR?